public/app/core/logs_model.ts

```diff
@@ -311,6 +314,8 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefi
     for (let j = 0; j < series.length; j++) {
       const ts = timeField.values.get(j);
       const time = dateTime(ts);
+      const tsNs = isLoki ? timeNanosecondField.values.get(j) : undefined;
+      const timeEpochNs = Number(isLoki ? tsNs : time.valueOf() + '000000');
```

Comment:
Well, technically we don't have to add padding from the right there, as on a millisecond precision all the timeEpochNs values are just going to be shorter, but still wanted to make sure all nanosecond-level timestamps have the same structure.